rfc6962-bis.xml

```diff
@@ -1455,10 +1455,7 @@ but it is expected there will be a variety.
         </section>
         <section title="TLS Feature Extension">
           <t>
-            If any certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension, then CT compliance (using any of the mechanisms from <xref target="tls_servers"/>) is required.
-          </t>
-          <t>
-            TLS clients MUST treat certificates which fail this requirement as non-compliant.
+            If any certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension, then CT compliance (using any of the mechanisms from <xref target="tls_servers"/>) is required, and TLS clients MUST refuse the connection if the chain is not compliant.
```

Comment:
I agree that the "TLS clients MUST treat certificates which fail this requirement as non-compliant" sentence is redundant, because we already say "...then CT compliance...is required".

However, the "TLS clients MUST refuse the connection" part belongs in the "Handling of Non-compliance" section, not here.